pkg/scheduler/scheduler.go

```diff
@@ -675,7 +675,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		} else {
 			// Calculating nodeResourceString can be heavy. Avoid it if klog verbosity is below 2.
 			if klog.V(2).Enabled() {
-				klog.Infof("pod %v/%v is bound successfully on node %q, %d nodes evaluated, %d nodes were found feasible.", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodes, scheduleResult.FeasibleNodes)
+				klog.InfoS("pod is bound successfully on node", "pod", klog.KRef(pod.Namespace, pod.Name), "node", scheduleResult.SuggestedHost, "evaluated nodes", scheduleResult.EvaluatedNodes, "feasible nodes", scheduleResult.FeasibleNodes)
```

Comment:
```suggestion
				klog.InfoS("Succesfully bound pod to node", "pod", klog.KObj(pod), "host", scheduleResult.SuggestedHost, "evaluatedNodes", scheduleResult.EvaluatedNodes, "feasibleNodes", scheduleResult.FeasibleNodes)
```
Please use active voice and lowerCamelCase.
Can you check what type is SuggestedHost?
* If it's a node object we should use klog.KObj(scheduleResult.SuggestedHost). 
* If it's a string with node name then we should use klog.KRef("", scheduleResult.SuggestedHost.Name)
* If it's anything else ("host"!="node") then we should not use key "node", but "host"

Reply:
```
// ScheduleResult represents the result of one pod scheduled. It will contain
// the final selected Node, along with the selected intermediate information.
type ScheduleResult struct {
	// Name of the scheduler suggest host
	SuggestedHost string
	// Number of nodes scheduler evaluated on one pod scheduled
	EvaluatedNodes int
	// Number of feasible nodes on one pod scheduled
	FeasibleNodes int
}

```

Reply:
Weird that host is only used in this context. I guess that it's using node name, but would need confirmation familiar with scheduler code

Reply:
this is legacy, use node instead of host.

Reply:
ok